src/group_cmd.cpp

```diff
@@ -298,6 +298,7 @@ void PropagateChildLivery(const Group *g)
 Group::Group(Owner owner)
 {
 	this->owner = owner;
+	this->folded = false;
```

Comment:
Is this not `false` by default?

Reply:
i always get confused about when members are zeroed on initialisation or not. No harm in being explicit?

Reply:
No harm.